### DISCUSSION TITLE
Is follow mode not implemented yet?

### DISCUSSION THREAD

**Discussion by txtyash** [helix-editor/helix#1148] [General] [answered]
There's a github issue about built-in easymotion or jumpmode that has been closed without any response about whether it has been implemented or not.

If it is then what are the keybindings to use it?

**Comment by NNBnh** [comment, marked as answer]
Hi, i found #510, #274 are still open
and an issue #855 that had been closed for duplication in this reply https://github.com/helix-editor/helix/issues/855#issuecomment-944919321

**Reply by txtyash** [reply]
Oh I'm sorry I misread.

**Reply by txtyash** [reply]
Any progress on it? When is it going to be implemented?

**Reply by sudormrfbin** [reply]
It's blocked by #417, but @Omnikar has already started working on it.

**Reply by Omnikar** [reply]
I've opened a draft PR with my progress: #1162
